src/types/markers.js

```diff
@@ -176,6 +176,12 @@ export type GCMinorCompletedData = {
 
   // The size of the data moved into the tenured heap.
   bytes_tenured: number,
+  // The number of cells tenured (not in older profiles).
+  cells_tenured?: number,
+
+  // The numbers of cells allocated since the previous minor GC.
+  cells_allocated_nursery?: number,
+  cells_allocated_tenured?: number,
```

Comment:
these ones are new as well, you can add a link to the bugzilla bug # that introduced them too.